ninafw_reset.go

```diff
@@ -0,0 +1,15 @@
+//go:build ninafw && !ninafw_reset_inverse
```

Comment:
What about making this a constant in the machine package?
That way you can basically do:

```go
if machine.NINA_RESET_INVERSE {
    // ...
} else {
    // ...
}
```

I think such code is easier to read and doesn't lead to extra clutter (the list of files is already pretty long).

Reply:
Good idea. Done.